src/htm/encoders/RandomDistributedScalarEncoder.cpp

```diff
@@ -118,14 +120,30 @@ void RandomDistributedScalarEncoder::encode(Real64 input, SDR &output)
     // deviations in the sparsity or semantic similarity, depending on how
     // they're handled.
 
-    // Exercise for the reader: Calculate the probability of a hash collision
-    // and account for it in the sparsity.
```

Comment:
never liked this! :+1:  
so..were the collisions more common than we expected (in settings people actually used)?

Reply:
HTM systems require a lot of cells in order to work properly.

I think there is a tendency for people (myself included) to make small HTM systems, either because
- it runs faster, 
- or they think it will be easier to understand/debug (in this case you're better off using a regular scalar encoder).

And even though I understand all this, I'm still tempted to reduce the number of cells in my experiments because I'm impatient and want it to run faster.